guillotina/contrib/dbusers/content/groups.py

```diff
@@ -0,0 +1,57 @@
+from guillotina import configure
+from guillotina import schema
+from guillotina.content import Folder
+from guillotina.contrib.dbusers import _
+from guillotina.interfaces import IFolder
+from zope.interface import implementer
+
+import typing
+
+
+class IGroupManager(IFolder):
+    pass
+
+
+class IGroup(IFolder):
+
+    name = schema.TextLine(title=_("Group name"), required=False)
+    description = schema.TextLine(title=_("Group Description"), required=False)
+    user_roles = schema.List(
+        title=_("Roles"), value_type=schema.TextLine(), required=False
+    )
+    users = schema.List(
+        title=_("Users"), value_type=schema.TextLine(), required=False, default=[]
+    )
+
+
+@configure.contenttype(
+    type_name="Group",
+    schema=IGroup,
+    add_permission="guillotina.AddGroup",
+    behaviors=["guillotina.behaviors.dublincore.IDublinCore"],
+)
+class Group(Folder):
```

Comment:
I guess if `User` inherigs from `IPrincipal`, then `Group` should do aswell?

Do you remember why that was needed?

Reply:
not needed.